.graphqlconfig.yaml

```diff
@@ -0,0 +1,42 @@
+projects:
+  users:
+    schemaPath: schemas/users.json
+    includes:
+      - '**/*.users.graphql'
+    extensions:
+      endpoints:
+        production:
+          url: 'https://mc-api.commercetools.com/graphql'
+          headers:
+            X-Graphql-Target: mc
+  menus:
```

Comment:
Also wondering. Maybe appShell?

Reply:
Hmm or maybe `proxy`? However the Schema is about menu configs, so I think it’s fine to keep it that way for now

Reply:
I think for this menus is fine as it is not a GraphQL target. Still to be honest I would not mind proxy. Also feels like a good name as we name these as to the service they hit less to what content they return.